lib/config/prettier.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: ['plugin:shopify/esnext', 'prettier'],
+
+  plugins: ['prettier'],
+
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true,
+        trailingComma: 'es5',
```

Comment:
Hmm, I think we need to change this for the ESNext config — any way of making this dynamic based on other configs passed in?

Reply:
The prettier config will [disable](https://github.com/prettier/eslint-config-prettier/blob/master/index.js#L22-L24) the rule(s) regardless of the config passed in.

Extending this prettier config means you opt-in to the opinionated prettier experience. If a consumer wants, they can update the `prettier/prettier` rules in their config. 

```
    'prettier/prettier': [
      'error',
      {
        singleQuote: true,
        trailingComma: '<none|es5|all>',
        bracketSpacing: false,
        jsxBracketSameLine: false,
      },
    ],
```